Add to FAQ known cases where mirrord can't load. (#41)
metalbear-co/mirrord.dev: content/docs/overview/faq.md

Yes. However, traffic mirroring isn't currently supported - you can use the --steal argument to steal traffic instead.

> ## What if my program just executes locally and there is no effect by mirrord?

Reviewer: ```suggestion
## I've run my program with mirrord but it seems to have no effect, what could be the issue?
```

> There are two known cases where mirrord currently cannot load into the application's process.

Reviewer: ```suggestion
There are currently two known cases where mirrord cannot load into the application's process:
```

1. [SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection) on Mac. Check the logs for a warning about SIP.

>    mirrord currently can't load into SIP binaries (we're working on it - coming soon). In the meanwhile you could try

Reviewer: ```suggestion
   mirrord currently can't load into SIP binaries (we're working on a fix). In the meanwhile you could try
```

   copying the binary you're trying to run into an unprotected directory (e.g. anywhere in your home directory) and

>    if necessary also remove the signature with `sudo codesign --remove-signature ./<your-binary>`.

Reviewer: ```suggestion
   if it still doesn't work, also remove the signature with `sudo codesign --remove-signature ./<your-binary>`.
```

> 2. Statically linked binaries. Since mirrord is using the dynamic linker to load into the application's process and

Reviewer: ```suggestion
2. Statically linked binaries. Since mirrord uses the dynamic linker to load into the application's process, 
```

>    hook some key functions, it cannot load if the binary is statically linked. Support for statically linked

Reviewer: ```suggestion
   it cannot load if the binary is statically linked. Support for statically linked
```

   binaries is planned for the long term, but for now you would have to make sure your binaries are dynamically

>    linked in order to use mirrord on them. With Go programs, for example, it is as simple as adding `import "C"` to

Reviewer: ```suggestion
   linked in order to run them with mirrord. With Go programs, for example, it is as simple as adding `import "C"` to
```
